azurerm/resource_arm_api_management.go

```diff
@@ -77,15 +80,28 @@ func resourceArmApiManagementService() *schema.Resource {
 								string(apimanagement.SkuTypePremium),
 							}, false),
 						},
+
 						"capacity": {
 							Type:         schema.TypeInt,
 							Required:     true,
-							ValidateFunc: validation.IntAtLeast(0),
+							ValidateFunc: validation.IntAtLeast(1),
```

Comment:
How come this value changed?

Reply:
Per the SDK implementation:
```
// Capacity - Capacity of the SKU (number of deployed units of the SKU). The default value is 1.
```

Reply:
that says default not minimum? thou it does make sense

Reply:
Reverted code and made it like it was, to avoid potential impact to existing users...